blog_system/settings.py

```diff
@@ -0,0 +1,134 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 5.1.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-$i#r(6%639re&%iogwn@^xz!ui@gw6&sw-f-^#b+3jf0&qfn&h"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True

```

Comment:
The `DEBUG` setting is set to `True`. This should be set to `False` in production to prevent sensitive information from being exposed in error pages.